apps/system/utils/netcmd.c

```diff
@@ -604,12 +606,14 @@ const static tash_cmdlist_t net_appcmds[] = {
 #ifdef CONFIG_SYSTEM_NETDB
 	{"netdb", netdb_main, TASH_EXECMD_SYNC},
 #endif
+#ifdef CONFIG_EXTERNAL_WAKAAMA_EXAMPLE
```

Comment:
Isn't the extra check redundant? If indeed, external wakaama example wasn't enabled, CONFIG_LWM2M_CLIENT_MODE would not be defined. So the build would never include the lwm2m_client_main function.